packages/react-redux/src/slices/cartoSlice.js

```diff
@@ -233,6 +250,16 @@ export const removeFilter = ({ id, column }) => ({
   payload: { id, column }
 });
 
+/**
+ * Action to remove filter by widget owner
+ * @param {id} - sourceId of the filter to remove
+ * @param {owner} - widgetId of the filter to remove
+ */
+export const removeFilterByOwner = ({ id, owner }) => ({
```

Comment:
Owner is the internal name, but I would probably suggest something more clear for the end user, like 'removeFilterFromWidget ({ sourceId, widgetId })

Reply:
Also, I'm wondering if we need the sourceId, or if we can just iterate on all sources, without asking for the param, Does this make sense?

Can a widget affect several sources? I don't think so... right?

Reply:
You're right. I did it like this because it made my life easier for the integration with builder but it would be better to remove it

Reply:
Thinking better of it, this action is not necessary. Removed